### PR TITLE
3.2.2 release

### PR DESCRIPTION
**Pull request by wing328**
### PR checklist

- [ ] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [ ] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [ ] Filed the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `3.3.x`, `4.0.x`. Default: `master`.
- [ ] Copied the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

(details of the change, additional tests that have been done, reference to the issue for tracking, etc)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -484,7 +484,7 @@ Here are some companies/projects (alphabetical order) using OpenAPI Generator in
...
 - 2018/06/21 - [Connect your JHipster apps to the world of APIs with OpenAPI and gRPC](https://fr.slideshare.net/chbornet/jhipster-conf-2018-connect-your-jhipster-apps-to-the-world-of-apis-with-openapi-and-grpc) by [Christophe Bornet](https://github.com/cbornet) at [JHipster Conf 2018](https://jhipster-conf.github.io/)
-- 2018/06/27 - [Lessons Learned from Leading an Open-Source Project Supporting 30+ Programming Languages](https://speakerdeck.com/wing328/lessons-learned-from-leading-an-open-source-project-supporting-30-plus-programming-languages) - [William Cheng](https://github.com/wing328) at [LinuxCon + ContainerCon + CloudOpen China 2018](https://www.lfasiallc.com/events/lc3-2018/)
+- 2018/06/27 - [Lessons Learned from Leading an Open-Source Project Supporting 30+ Programming Languages](https://speakerdeck.com/wing328/lessons-learned-from-leading-an-open-source-project-supporting-30-plus-programming-languages) - [William Cheng](https://github.com/wing328) at [LinuxCon + ContainerCon + CloudOpen China 2018](https://www.lfasiallc.com/events/lc3.2.28/)
```
Comment (jmini) [inline README.md]: maybe this should be reverted.

Comment (wing328) [reply README.md]: Thx. fixed via https://github.com/OpenAPITools/openapi-generator/pull/869/commits/e95f15c5c53b38ac5399ceecb90599b0df3a8c17 using the bitly link

**Comment by jmini** [comment]
Tag created (with release notes):
https://github.com/OpenAPITools/openapi-generator/releases/tag/v3.2.2

**Comment by jmini** [comment]
PR to prepare next version: OpenAPITools/openapi-generator#870

**Comment by jmini** [comment]
Check with `ruby bin/utils/release_checkout.rb 3.2.2`:

```
Running checkout on OpenAPI Generator release 3.2.2
Checking homebrew forumla ... [ERROR]
> https://raw.githubusercontent.com/Homebrew/homebrew-core/master/Formula/openapi-generator.rb not yet updated with https://search.maven.org/remotecontent?filepath=org/openapitools/openapi-generator-cli/3.2.2/openapi-generator-cli-3.2.2.jar
Checking openapi-generator JAR ... [OK]
Checking openapi-generator-cli JAR ... [OK]
Checking openapi-generator-maven-plugin JAR ... [OK]
Checking openapi-generator-gradle-plugin JAR ... [ERROR]
> http://central.maven.org/maven2/org/openapitools/openapi-generator-gradle-plugin/3.2.2/openapi-generator-gradle-plugin-3.2.2.jar not found
Checking openapi-generator-online JAR ... [OK]
Checking openapi-generator-project pom.xml ... [OK]
Checking openapi-generator README.md ... [OK]
Checking openapi-generator-online docker ... [OK]
Checking openapi-generator-cli docker ... [OK]
```

**Comment by jmini** [comment]
https://github.com/OpenAPITools/openapi-generator/pull/870 is merged =>`master` can be used for `3.2.3-SNAPSHOT`

**Comment by jmini** [comment]
Gradle published on Sonatype releases: [3.2.2](https://oss.sonatype.org/content/repositories/releases/org/openapitools/openapi-generator-gradle-plugin/3.2.2/)

```
https://oss.sonatype.org/content/repositories/releases
```

It will appear on maven central soon.

**Comment by jmini** [comment]
Release Twitt by @wing328 : https://twitter.com/oas_generator/status/1032252335131512832

**Comment by jmini** [comment]
Milestone [3.2.2](https://github.com/OpenAPITools/openapi-generator/milestones/12) closed

**Comment by jmini** [comment]
Homebrew: Homebrew/homebrew-core#31385

**Pull request merged**